studio/src/ui/popoverMenus/popoverMenus.css

```diff
@@ -36,47 +36,61 @@
 	stroke-width: 1px;
 }
 
-.contextMenuItem{
-	padding: 1px 8px;
+button.context-menu-item {
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	width: 100%;
+	border: none;
+	padding: 2px 8px;
+	background-color: transparent;
 }
-.contextMenuItemText {
-	display: inline;
+button.context-menu-item[disabled] {
+	color: var(--default-text-color-disabled-context-menu);
 }
-.contextMenuItemIcon {
-	display: inline-block;
+
+.context-menu-item-content {
+	display: flex;
+	align-items: center;
+	gap: 4px;
+}
+
+.context-menu-item-icon {
 	width: 20px;
 	height: 1em;
 	background-size: contain;
-	background-repeat: no-repeat;
-	vertical-align: text-top;
+	background-repeat:no-repeat;
+	vertical-align: middle;
```

Comment:
Hah nice catch! I didn't even realise these were never aligned.
I don't think `vertical-align: middle` is necessary anymore now that it's using flexbox though.

Reply:
Hmm when I toggle it on and off in DevTools I don't see any noticable difference. Both `.context-menu-item` and `.context-menu-item-content` have `display: flex`.